config/initializers/date_formats.rb

```diff
@@ -23,12 +23,7 @@
              '%e %B %Y at %l:%M%P'
            end
 
-  time.strftime(format).squish
-end
-
-Time::DATE_FORMATS[:govuk_time_and_date] = lambda do |time|
```

Comment:
We weren't using this anywhere. We also have a `govuk_date_and_time` method which _is_ used.